modules/platform/nuxeo-platform-userworkspace/nuxeo-platform-userworkspace-core/src/test/java/org/nuxeo/ecm/platform/userworkspace/core/tests/TestUserWorkspace.java

```diff
@@ -146,8 +147,8 @@ public void testMultiDomains() {
         assertTrue(uw.getPathAsString(), uw.getPathAsString().startsWith("/default-domain"));
 
         // now delete the default-domain
+        txFeature.nextTransaction();
```

Comment:
wouldn't you want to swap those two lines ? So the comment is right above the `session.removeDocument()` ?

Reply:
Yes you're right, and I'll add a comment explaining why we're calling `nextTransaction()`.